src/main/java/hudson/plugins/ansicolor/AnsiColorBuildWrapper.java

```diff
@@ -86,31 +99,35 @@ public Environment setUp(AbstractBuild build, Launcher launcher,
 	public OutputStream decorateLogger(AbstractBuild build, final OutputStream logger) {
 		final AnsiColorMap colorMap = getDescriptor().getColorMap(getColorMapName());
 
-        return new LineTransformationOutputStream() {
-            AnsiHtmlOutputStream ansi = new AnsiHtmlOutputStream(logger, colorMap, new AnsiAttributeElement.Emitter() {
-                public void emitHtml(String html) {
-                    try {
-                        new SimpleHtmlNote(html).encodeTo(logger);
-                    } catch (IOException e) {
-                        LOG.log(Level.WARNING, "Failed to add HTML markup '" + html + "'", e);
-                    }
-                }
-            });
-
-            @Override
-            protected void eol(byte[] b, int len) throws IOException {
-                ansi.write(b, 0, len);
-                ansi.flush();
-                logger.flush();
-            }
-
-            @Override
-            public void close() throws IOException {
-                ansi.close();
-                logger.close();
-                super.close();
-            }
-        };
+		if (logger != null)
```

Comment:
Make this a bit more readable with some `{}` or maybe even with `if (logger == null) { return ... }`